XenAdmin/Diagnostics/Problems/VMProblem/AutoStartEnabled.cs

```diff
@@ -58,7 +58,7 @@ public override string Description
         protected override AsyncAction CreateAction(out bool cancelled)
         {
             cancelled = false;
-            return new DelegatedAsyncAction(_connection, string.Format(Messages.DISABLED_AUTOSTART_ON_VM, Helpers.GetName(VM)), "", null, ActionDelegate(false));
+            return new DelegatedAsyncAction(_connection, Messages.ACTION_DISABLE_AUTOSTART_ON_VM, "", string.Format(Messages.ACTION_DISABLING_AUTOSTART_ON_VM, Helpers.GetName(VM)), ActionDelegate(false));

```

Comment:
Please use ACTION_DISABLING_AUTOSTART_ON_VM for the `startDescription` parameter (the 3rd parameter).

Same for the enabling action delegate.